source/channels/make-calls.rst

```diff
@@ -28,16 +27,59 @@ Starting from Mattermost v7.7 you can:
 
 Mattermost Cloud users can start calling right out of the box. For Mattermost self-hosted deployments, System Admins need to enable the plugin and adjust configurations `in the System Console </configure/plugins-configuration-settings.html#calls>`_.
 
-To start a call, select **Start call** in the channel header. Any active team member in the channel can join a call, whether it's a public or private channel. If someone from outside of the organization wants to join a call, you'll need to provide them with a guest account, and add them to the channel. Users who are archived or not registered can't join a call.
+Start a call
+------------
+
+.. tabs::
+
+  .. tab:: Desktop
+
+    To start a call, select **Start call** in the channel or message header. When you start a call in a channel, you're muted by default. In a direct or group message you're unmuted by default.
```

Comment:
For completeness I should mention that `/call start` or `/call join` slash commands can be used as well. Or even easier through the keyboard shortcut. That said, I am okay if we only want to keep the most common way here.